content/utility/services.cc

```diff
@@ -382,7 +382,11 @@ void RegisterMainThreadServices(mojo::ServiceFactory& services) {
   services.Add(RunDataDecoder);
   services.Add(RunStorageService);
   services.Add(RunTracing);
+  // Ideally the whole VideoCapture service would be gated behind a Chrome
```

Comment:
This lgtm. In the future i'm assuming we would fully overload this somewhere in our own content shell ?

Reply:
Sadly this seems to be part of content/utility and not part of content_shell. 

I might take a stab upstream at hiding all the video capture service behind a flag
(I'm still an OWNER so I guess I feel responsible :) - I should have caught this
during review.

Reply:
Apropos nothing - seems like RunAuctionWorkletService right above here is also entirely extraneous for us.

Reply:
Same for `RunDataDecoder()`. I'll disable those in later patches.